osf/metadata/schemas/datacite.json

```diff
@@ -0,0 +1,607 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "datacite-v4.0.json",
+  "title": "DataCite v4.0",
+  "description": "JSON representation of the DataCite v4.0 schema.",
```

Comment:
Just curious, where did we get this JSON representation and how did we decide on this schema?